### ISSUE TITLE
Spotify Web Player 5 song skip bug

### ISSUE THREAD

**Issue by OwlCacone** [focus-browser/browser#191]
After a bit of googlin it seems that this issue (which skips a couple songs in any given album or playlist and always plays the 6th one) resolved itself for multiple users 2 years ago after a browser update. Anyone else experienced this?

Btw, none of the other fixes worked for me (re-logging back in, clearing browser cache, cookies)

**Comment by trickypr** [comment]
I just created a spotify account to test this. Can I confirm that the following reproduction instructions seem correct?
1. Open spotify (`open.spotify.com`)
2. Select a random playlist (e.g. `https://open.spotify.com/playlist/37i9dQZF1DWZeKCadgRdKQ`)
3. Click play.
4. Spotify will skip 3-5 songs and stay paused.

**Comment by trickypr** [comment]
**Update:** My network config seems to be blocking parts of spotify.

@OwlCacone Can you confirm that spotify works if you use it on Firefox?

**Comment by OwlCacone** [comment]
@trickypr Yes, I tried using the web player on Firefox and it worked without a hitch

**Comment by SoaringGecko** [comment]
Same here skips like 5 songs and gets stuck on Pulse but works fine on Brave, Firefox, Waterfox, and Chrome. I do want to point out this happens no matter what you are listening to, not just playlists. Spotify iframes also appear to take up the space they normally would but don't actually load.


![image](https://github.com/pulse-browser/browser/assets/48862796/59438b61-047e-448c-9614-3ab3dfb4fef7)
![image](https://github.com/pulse-browser/browser/assets/48862796/6859d09a-16e3-4498-a120-2bd397bd6111)

**Comment by trickypr** [comment]
Note to self: This can be reproduced on Windows. This corresponds with some widevide license errors.
